code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -374,79 +374,32 @@
 	id = /datum/reagent/medicine/stimpak
 	results = list(/datum/reagent/medicine/stimpak = 1)
 	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/medicine/spaceacillin = 1)
-/*	OptimalTempMin 		= 500 // Lower area of bell curve for determining heat based rate reactions
-	OptimalTempMax		= 550 // Upper end for above
-	ExplodeTemp			= 9999 //Temperature at which reaction explodes
-	OptimalpHMin		= 3 // Lowest value of pH determining pH a 1 value for pH based rate reactions (Plateu phase)
-	OptimalpHMax		= 8 // Higest value for above
-	ReactpHLim			= 4 // How far out pH wil react, giving impurity place (Exponential phase)
-	CurveSharpT 		= 5 // How sharp the temperature exponential curve is (to the power of value)
-	CurveSharppH 		= 0.5 // How sharp the pH exponential curve is (to the power of value)
-	ThermicConstant		= -6 //Temperature change per 1u produced
-	HIonRelease 		= -0.1 //pH change per 1u reaction
-	RateUpLim 			= 5 //Optimal/max rate possible if all conditions are perfect
-	FermiChem 			= TRUE//If the chemical uses the Fermichem reaction mechanics
-	FermiExplode 		= FALSE //If the chemical explodes in a special way
-*/
-
-/datum/chemical_reaction/stimpak2
+
+/datum/chemical_reaction/stimpak_imitation
 	name = "Imitation Stimpak Fluid"
 	id = /datum/reagent/medicine/stimpakimitation
 	results = list(/datum/reagent/medicine/stimpakimitation = 2)
 	required_reagents = list(/datum/reagent/consumable/brocjuice = 1, /datum/reagent/consumable/xanderjuice = 1)
-/*	OptimalTempMin 		= 500 // Lower area of bell curve for determining heat based rate reactions
-	OptimalTempMax		= 650 // Upper end for above
-	ExplodeTemp			= 9999 //Temperature at which reaction explodes
-	OptimalpHMin		= 2 // Lowest value of pH determining pH a 1 value for pH based rate reactions (Plateu phase)
-	OptimalpHMax		= 8 // Higest value for above
-	ReactpHLim			= 4 // How far out pH wil react, giving impurity place (Exponential phase)
-	CurveSharpT 		= 5 // How sharp the temperature exponential curve is (to the power of value)
-	CurveSharppH 		= 0.5 // How sharp the pH exponential curve is (to the power of value)
-	ThermicConstant		= -6 //Temperature change per 1u produced
-	HIonRelease 		= -0.1 //pH change per 1u reaction
-	RateUpLim 			= 12 //Optimal/max rate possible if all conditions are perfect
-	FermiChem 			= TRUE//If the chemical uses the Fermichem reaction mechanics
-	FermiExplode 		= FALSE //If the chemical explodes in a special way
-*/
+
+/datum/chemical_reaction/stimpak/synthetic
+	id = "stimpak_synthetic"
+	required_reagents = list(/datum/reagent/blood/synthetics = 1, /datum/reagent/medicine/spaceacillin = 1)
```

Comment:
No results / name?

Reply:
It's inherited by the /stimpak
Kind of like 
```
/datum/chemical_reaction/synthflesh
	name = "Synthflesh"
	id = /datum/reagent/medicine/synthflesh
	results = list(/datum/reagent/medicine/synthflesh = 3)
	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/carbon = 1, /datum/reagent/medicine/styptic_powder = 1)
```
	
```
/datum/chemical_reaction/synthflesh/synthblood
	id = "synthflesh_2"
required_reagents = list(/datum/reagent/blood/synthetics = 1, /datum/reagent/carbon = 1, /datum/reagent/medicine/styptic_powder = 1)
```

Reply:
It makes the code look a little cleaner, which is why I choose to do it.
And it worked fine in the test for the use of synth blood to stimpak